.travis.yml

```diff
@@ -2,7 +2,6 @@
 language: ruby
 
 rvm:
-  - 1.9.3
```

Comment:
I removed this commit and will make a new separate PR to remove the unsupported Puppet Versions and update the documentation.